GenericSerialDeviceRemoteControl/GenericSerialDeviceRemoteControl.py

```diff
@@ -0,0 +1,350 @@
+from __main__ import vtk, slicer
+from slicer.ScriptedLoadableModule import ScriptedLoadableModule, ScriptedLoadableModuleWidget, ScriptedLoadableModuleLogic
+from slicer.util import VTKObservationMixin
+import logging
+
+
+class GenericSerialDeviceRemoteControl(ScriptedLoadableModule):
+    def __init__(self, parent):
+        ScriptedLoadableModule.__init__(self, parent)
+        self.parent.title = "Generic Serial Device Remote Control"
```

Comment:
GenericSerialDevice is a good class name, but for a name that users see, I would skip the `Generic`. The module name would then be simply `Serial Device Remote Control`.

Reply:
Yeah I was just about to an include a comment about the same thing.
![image](https://user-images.githubusercontent.com/15837524/187506619-f32acd14-62e1-4dc0-947d-62bb2ee78111.png)

^ A module icon for it would also be nice

Reply:
Any thoughts on what the module icon would be? Is just copying the OpenIGTLink Remote icon fine?

Reply:
I don’t think duplicating the icon of another module would add much value beyond the default extension icon.

how about something based on the serial nature such as 0/1s along a single line between two endpoints? Making the point it is serial and not parallel streams.

Reply:
A serial port icon could work, too - like this:

https://iconscout.com/icon/serial-port-2653340